app/src/main/java/com/project/symptoms/view/BodyView.java

```diff
@@ -198,12 +189,9 @@ private Rect calculateCenteredBounds() {
 
         int sideMargin = (int) (remainingFreeWidth / 2);
 
-        int left = sideMargin;
         int right = 3*sideMargin;
         int top = 0;
```

Comment:
I think that separating the four bounds variables from where they value come from is more readable (as it was before)

Reply:
The IDE it self showed a warning that is unnecessary to have that. But I can change it.